external/fv3fit/fv3fit/reservoir/train.py

```diff
@@ -138,81 +140,85 @@ def train_reservoir_model(
         BatchLinearRegressor(hyperparameters.readout_hyperparameters)
         for r in range(rank_divider.n_subdomains)
     ]
-    sync_tracker = SynchronziationTracker(
-        n_synchronize=hyperparameters.n_timesteps_synchronize
-    )
-    for b, batch_data in enumerate(train_batches):
-        input_time_series = process_batch_data(
-            variables=hyperparameters.input_variables,
-            batch_data=batch_data,
-            rank_divider=rank_divider,
-            autoencoder=transformers.input,
-            trim_halo=False,
-        )
-        # If the output variables differ from inputs, use the transformer specific
-        # to the output set to transform the output data
-        _output_rank_divider_with_overlap = rank_divider.get_new_zdim_rank_divider(
-            z_feature_size=transformers.output.n_latent_dims
-        )
-        output_time_series = process_batch_data(
-            variables=hyperparameters.output_variables,
-            batch_data=batch_data,
-            rank_divider=_output_rank_divider_with_overlap,
-            autoencoder=transformers.output,
-            trim_halo=True,
-        )
 
-        # reservoir increment occurs in this call, so always call this
-        # function even if X, Y are not used for readout training.
-        reservoir_state_time_series = _get_reservoir_state_time_series(
-            input_time_series, hyperparameters.input_noise, reservoir
+    for train_batches in train_batches_sequence:
```

Comment:
All the code that iterates over a time series' batches within this loop is the same as before.